### DISCUSSION TITLE
GraphQL auth user field

### DISCUSSION THREAD

**Discussion by NickSettler** [directus/directus#10720] [Feature Requests]
There's an idea of adding user field for GraphQL auth mutation. And maybe also for RestAPI. When the user signs in via RestAPI or GraphQL he recieves only tokens and token expiration date. Maybe there would be possible to add the user object itself for the auth query?

**Comment by rijkvanzanten** [comment]
What would be the benefit over using users_me (besides having to make one request less)?

**Reply by NickSettler** [reply]
To store all information about user after one request not making another one. It can reduce impact on server and increase speed for the client - no need to wait for another request that might take a longer time than even the first

**Reply by rijkvanzanten** [reply]
True! Though the only benefit would be the reduction in network latency overhead, the impact on the server should be nearly identical, as it would be a different query flow in the API anyways

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
